src/convertToObject.js

```diff
@@ -6,7 +7,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .reduce((prev, el) => {
+      const [property, value] = el
+        .split(':')
+        .map((elem) => elem.trim());
+
+      prev[property] = value;
```

Comment:
The code assigns values to the `prev` object without checking if `property` is not an empty string. This can result in adding unwanted keys to the object. You should add a condition to check if `property` is a non-empty string before assigning it to `prev`.